### PR TITLE
Added maphacks for handsoap in the store in z2a4

### PR DESCRIPTION
**Pull request by fhomolka**
Here's the maphacks I promised to try
I've added the maphack include at the end of voxels.def, I hope that's okay

### CODE REVIEW

File: maphacks/z2a4.mhk [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Handsoap Bottles in the perfume store
+
+// Left
+sprite 2262 angoff 1536
```
Comment (Dzierzan) [inline maphacks/z2a4.mhk]: I'd add comment to every line with voxel's name. Take a look at Duke Nukem voxel pack.

File: voxels.def [hunk 2]
```diff
@@ -328,4 +328,6 @@ voxel "voxels/3657_VALVE2_G.kvx"		{ tile 3657 }
 voxel "voxels/3658_VALVE2_H.kvx"		{ tile 3658 }
 
 voxel "voxels/3659_SOCKET1.kvx"			{ tile 3659 }
-voxel "voxels/3660_SOCKET2.kvx"			{ tile 3660 }
\ No newline at end of file
+voxel "voxels/3660_SOCKET2.kvx"			{ tile 3660 }
+
+include maphacks.def
```
Comment (Dzierzan) [inline voxels.def]: I would add `**include**` on top of everything here.

File: maphacks/z2a4.mhk (hunk 1 above)
Comment (fhomolka) [reply maphacks/z2a4.mhk]: Added descriptions to each line

File: voxels.def (hunk 2 above)
Comment (fhomolka) [reply voxels.def]: moved include on top. Wasn't sure if voxels had to be loaded prior

**Comment by Dzierzan** [comment]
LGTM. Just squash the commits and I'll merge it.

**Comment by fhomolka** [comment]
Okay, I did some stupid things in the process, but they should be squashed now

**Pull request merged**